### DISCUSSION TITLE
my each is not work

### DISCUSSION THREAD

**Discussion by HanxuGit** [handlebars-lang/handlebars.js#2045] [Q&A] [answered]
Back-end java language
<img width="790" alt="image" src="https://github.com/user-attachments/assets/bf462ef1-6247-4188-a9e0-bd7f25bbde10">
fronted page alertEmail.hbs 
<img width="685" alt="image" src="https://github.com/user-attachments/assets/b86bbfb8-b470-416b-9421-14b898b50aac">
each is not work

**Comment by jaylinski** [comment]
Do you get an error? I need a little bit more information in order to help you.

**Reply by HanxuGit** [reply]
I pass a list parameter to the backend, but the front-end cannot iterate through it using the #each directive and no errors are displayed.

**Reply by HanxuGit** [reply]
handlebars version	4.4.0

**Reply by jaylinski** [reply]
> I pass a list parameter to the backend, but the front-end cannot iterate through it using the #each directive and no errors are displayed.

This works fine with handlebars.js: https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22%7B%7B%23each%20riskAlertInfoList%7D%7D%5Cn%5Ct%7B%7BthresholdDesc%7D%7D%5Cn%7B%7B%2Feach%7D%7D%5Cn%22%2C%22partials%22%3A%5B%5D%2C%22input%22%3A%22%7B%5Cn%20%20riskAlertInfoList%3A%20%5B%5Cn%20%20%20%20%7B%5Cn%20%20%20%20%20%20thresholdDesc%3A%20%5C%22foo%5C%22%5Cn%20%20%20%20%7D%2C%5Cn%20%20%20%20%7B%5Cn%20%20%20%20%20%20thresholdDesc%3A%20%5C%22bar%5C%22%5Cn%20%20%20%20%7D%5Cn%20%20%5D%5Cn%7D%5Cn%22%2C%22output%22%3A%22%5Ctfoo%5Cn%5Ctbar%5Cn%22%2C%22preparationScript%22%3A%22%5Cn%22%2C%22handlebarsVersion%22%3A%224.7.8%22%7D

I guess there is something wrong with your setup. Since you are using the Java implementation, you can ask here: https://github.com/jknack/handlebars.java

**Reply by HanxuGit** [reply, marked as answer]
iThe problem has been resolved, thank you for your response.
